product-mini/platforms/zephyr/simple-http/wasm-apps/http_get.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <errno.h>
+#include <string.h>
+
+#ifdef __wasi__
+#include <wasi_socket_ext.h>
+#endif
+
+/* HTTP server to connect to */
+#define HTTP_HOST "192.0.2.10"
```

Comment:
This might be confusing since this constant is used in the LOG statement (line 42) and the actual request (line 31) but the actual IP to connect to is not derived from this string (see line 49).
In order to modify this example and to change from which HTTP server to read, it is not clear which lines need adaptation.
Same comment for PORT since the actual port used to connect is found in line 47

Reply:
added a configuration section in readme.